packages/common/src/PreventExit/hooks.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useMemo } from 'react';
+import { createPreventExitListener } from './store';
+
+/**
+ * Custom hook that allows you
+ * to access the PreventExit Context methods inside a component.
+ *
+ * Note : If this hook is called inside a component
+ * that is not a child of a PreventExit component,
+ * it will throw an error.
```

Comment:
Properly format the width of the documentation : it should break lines at `120` width. I suggest you add a custom ruler at 120 characters width

Reply:
This comment is fully changed.